data/json/recipes/armor/bespoke_armor/survivor.json

```diff
@@ -114,43 +89,31 @@
     "difficulty": 5,
     "time": "6 h",
     "autolearn": true,
-    "using": [ [ "sewing_kevlar", 100 ], [ "fabric_lycra", 20 ] ],
+    "using": [ [ "sewing_kevlar", 100 ], [ "fabric_lycra", 20 ], [ "fabric_nylon", 30 ], [ "fastener_large", 1 ], [ "clasps", 8 ] ],
```

Comment:
why more nylon than lycra? if the nylon is an other layer and the lycra is an inner layer shouldn't they be approximately equivalent materials wise.

Reply:
The Nylon layer is 0.3mm and the Lycra is 0.2mm so I thought 1.5 times seemed reasonable. :) I can change them if you like.

   - Shane